keyboards/treadstone32/keymaps/default/keymap.c

```diff
@@ -0,0 +1,186 @@
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       KC_LSFT,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  KC_LEFT,  KC_DOWN,    KC_UP,  KC_RGHT,  KC_LSFT,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       KC_LCTL,  KC_LALT,  KC_LGUI,  XXXXXXX,  XXXXXXX,  KC_MINS,    KC_RO,  KC_COMM,   KC_DOT,  KC_SSCT,
+  //`---------+---------+---------+---------+---------+---------+---------+---------+---------+---------'
+                                               _______,  _______
+  //                                        `---------|---------'
+  ),
+
+  [_ADJUST] = LAYOUT(
+  //,---------------------------------------------------------------------------------------------------.
+        KC_RST,  KC_LRST,  KC_KNRM,  KC_KSWP,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       KC_LTOG,  KC_LHUI,  KC_LSAI,  KC_LVAI,  XXXXXXX,  KC_MS_L,  KC_MS_D,  KC_MS_U,  KC_MS_R,  XXXXXXX,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       KC_LMOD,  KC_LHUD,  KC_LSAD,  KC_LVAD,  XXXXXXX,  KC_BTN1,  KC_BTN2,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+  //`---------+---------+---------+---------+---------+---------+---------+---------+---------+---------'
+                                               _______,  _______
+  //                                        `---------|---------'
+  )
+};
+
+uint16_t get_tapping_term(uint16_t keycode) {
+  switch (keycode) {
+    case KC_BSLO:
+      return TAPPING_LAYER_TERM;
+    case KC_SPRA:
+      return TAPPING_LAYER_TERM;
+    default:
+      return TAPPING_TERM;
+  }
+}
+
+int RGB_current_mode;
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+
+  bool result = false;
+  switch (keycode) {
+    case KANJI:
+      if (record->event.pressed) {
+        if (keymap_config.swap_lalt_lgui == false) {
+          register_code(KC_LANG2);
+        } else {
+          SEND_STRING(SS_LALT("`"));
+        }
+      } else {
+        unregister_code(KC_LANG2);
+      }
+      break;
+    #ifdef RGBLIGHT_ENABLE
+      //led operations - RGB mode change now updates the RGB_current_mode to allow the right RGB mode to be set after reactive keys are released
+      case RGB_MOD:
+          if (record->event.pressed) {
+            rgblight_mode(RGB_current_mode);
+            rgblight_step();
+            RGB_current_mode = rgblight_config.mode;
+          }
+        break;
+      case RGBRST:
+          if (record->event.pressed) {
+            eeconfig_update_rgblight_default();
+            rgblight_enable();
+            RGB_current_mode = rgblight_config.mode;
+          }
+        break;
+    #endif
+    default:
+      result = true;
+      break;
+  }
+
+  return result;
+}
+
+void matrix_init_user(void) {
```

Comment:
This needs to be:
```suggestion
void keyboard_post_init_user(void) {
```

Otherwise, it loads too early.